### PR TITLE
feat(frontend): Object Field Edit page created

### LINKED ISSUE #2159: Object Field Edit - Create page
Warning: this issue is part of a series of issues aiming to create the "Object Field Edition flow". This is issue number 1.

## Context

See #1769 

For this issue:
- Create an empty page with the route: `/settings/objects/[PLURAL_OBJECT_NAME]/[FIELD_NAME]`.
- Use the breadcrumb component from #1814.

## Figma

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=11383%3A58516&mode=design&t=EynOEhEsP06qxkYt-1

## Behavior

**GIVEN** I am on an Object Field Edit page
**THEN** I see a breadcrumb

### PR DESCRIPTION
**Pull request by Kanav-Arora**
Closes #2159 

Route setup completed
React Route added in `App.ts` with path in `SettingsPath.ts`

Please find below the screenshot

Screenshot 2023-10-24 at 9 16 53 PM

### CODE REVIEW

File: front/src/App.tsx [hunk 1]
```diff
@@ -106,6 +107,10 @@ export const App = () => {
                   path={SettingsPath.ObjectNewFieldStep2}
                   element={<SettingsObjectNewFieldStep2 />}
                 />
+                <Route
+                  path={SettingsPath.FieldEdit}
+                  element={<SettingsFieldEdit />}
```
Comment (charlesBochet) [inline front/src/App.tsx:112]: SettingsObjectFieldEdit

File: front/src/App.tsx [hunk 2]
```diff
@@ -106,6 +107,10 @@ export const App = () => {
                   path={SettingsPath.ObjectNewFieldStep2}
                   element={<SettingsObjectNewFieldStep2 />}
                 />
+                <Route
+                  path={SettingsPath.FieldEdit}
```
Comment (charlesBochet) [inline front/src/App.tsx:111]: ObjectFieldEdit

**Review by charlesBochet — APPROVED** [review]
Approved with minor comments :)

**Comment by Kanav-Arora** [comment]
> Approved with minor comments :)

Done

**Pull request merged**